.github/ISSUE_TEMPLATE/engineering-manager-onboarding.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Engineering Manager Onboarding
+about: A list of things to do as a new manager to become familiar with the nature of our engineering team and the context it exists in.
+---
+
+## Your second week
+
+You’ve done general people-ops and engineering team onboarding, and have casually met some team-members. Now it’s time to start onboarding as a new engineering manager.
+
+- [ ] If you’re reading this, you’re the first external engineering manager we’ve hired and your name is Adam. Yay! (If not, this will be _real_ awkward.) At Artsy we are trying to give new hires ownership over improving the onboarding documentation that they are faced with, in an effort to help the next new hire onboard more smoothly. As you are the first one, there is plenty of work to do in this area; in fact, this very document is nothing but a first draft, very much a work in progress. It will be your task to make this document into a substantially larger one, that draws from all the things you had to learn. You’re not on your own, though, your fellow peers will all be available to support you during this task. You got this!
+- [ ] The people-ops team should have signed you up for the various on-site manager trainings.
+- [ ] Your manager should have invited you to meetings relevant to managers.
+- [ ] Your manager should have scheduled 1:1s for you with the other engineering managers, including the CTO, and other leadership people.
+
+### Extended sprint rotation and meet & greet
+
+Unlike individual contributors, who rotate for a few sprints, engineering managers are expected to rotate across all teams. While your main responsibility will be to be a people manager, we expect anybody on the engineering team to understand roughly how we do things and what systems are in use at Artsy, but _mostly_ this will be a chance to work with all the engineering team members and product people such as PMs and designers. Currently there are 5 teams, so with sprints having a duration of 2 weeks that’s easily going to span your next 2 months.
```

Comment:
I think it's actually very important that the manager is familiar with our tech in order to help unblock people, facilitate discussions, and set expectations correctly. This is slightly different than the expectation for new hires. 😄 

I wonder if we can reword this to more explicitly call that out. Brainstorm idea: perhaps the goal at the end of the sprint rotation should be for the person to do a short presentation (to other managers? to everyone?) to summarize our stack and how all the different teams contribute to it.

Reply:
I like that objective 👍  I _think_ it would be good to present it to everyone, both to contribute to presence _and_ so there’s more people that can provide feedback about potential misunderstandings?